Src/Logic/Registration/Registration.ts

```diff
@@ -34,10 +35,12 @@ export default function generateRegistrationForm(
   }
 
   function onSignUpClick(secondStepValues: SecondStepValues) {
+    const currentDate=new Date();
     const weight: Weight = {
-      date: new Date(),
-      weight: firstStepValues.currentWeight,
+      date: currentDate,
```

Comment:
I feel the old approach on how the date was defined was more concise - but it's just a suggestion here

Reply:
ah all right, I missed the date is used later as well so this approach is better then ;)

Reply:
ok ;)